Testreglar/1.4.5/App/app-1.4.5a.json

```diff
@@ -0,0 +1,162 @@
+{
+	"namn": "App-1.4.5a Bilde av tekst er ikkje brukt unødvendig 2023",
+	"id": "app-1.4.5a",
+	"testlabId": 518,
+	"versjon": "1.0",
+	"type": "App",
+	"spraak": "nn",
+	"kravTilSamsvar": "<ul><li>Bilde av tekst kan visuelt tilpasses brukerens krav<ul><li>skrifttype, størrelse, farge og bakgrunn kan endres</li></ul></li><li>Bilde av tekst er brukt fordi en bestemt presentasjon av innholdet er avgjørende</li></ul>",
```

Comment:
Samma kommentar som for Nett. Få tydeligere frem at det er bruken av bilde av tekst som er sentral for kravet, dagens krav til samsvar har bare info om unntakene.